style.css

```diff
@@ -2399,20 +2393,7 @@ media_tag_4 {media_subcode:4}
 	background-image: url(style/images/fancy-loading.gif) !important;
 }
 media_tag_5 {media_subcode:5}
-/*-----------------------------------------------------------------------------------*/
-/*	PROGRESS BAR
-/*-----------------------------------------------------------------------------------*/
-.progress-bar li p {
```

Comment:
I have no idea what happened here.  It seems like it was part of my initial commit, so it *must* have been an accident.  I blame my laptop again.  Its mouse buttons are *part* of the touchpad and sometimes when I'm working, it'll interpret them as a touch, sending the cursor flying across the screen.  It's caused me to accidentally select and type-over / delete things before, and I have no other explanation for how this section was removed. :sweat_smile: